src/components/EnterpriseContractView/SecurityEnterpriseContractTab.tsx

```diff
@@ -43,9 +43,12 @@ const getResultsSummary = (ECs, ecLoaded) => {
 };
 
 export const SecurityEnterpriseContractTab: React.FC<
-  React.PropsWithChildren<{ pipelineRun: string }>
-> = ({ pipelineRun }) => {
-  const [ecResult, ecResultLoaded] = useEnterpriseContractResults(pipelineRun);
+  React.PropsWithChildren<{
+    pipelineRunName: string;
```

Comment:
Would it make sense to pass the full pipeline run object here rather than the metadata values?

I have no objections with either strategy, I was just wondering if there was a reason why we extract the values, then pass to the component instead of passing the full object and minimizing the props.

Reply:
Good point! Thank you